File: Q16/Q16.py
#Remove duplicate in an array
arr = [1, 2, 3, 2, 4, 5, 5, 6]

print("Original List:", arr)

n = len(arr)

i = 0
while i < n:
    j = i + 1
    while j < n:
        if arr[i] == arr[j]:
           
            arr.pop(j)
            n -= 1  
        else:
            j += 1
    i += 1

print("List after removing duplicates:", arr)

File: Q16/Q16.js
//Remove the duplicate in an array
let arr = [1, 2, 3, 2, 4, 5, 5, 6];

console.log("Original Array:", arr);

let n = arr.length;

for (let i = 0; i < n; i++) 
    {
    for (let j = i + 1; j < n; j++) 
        {
        if (arr[i] === arr[j]) 
        {
            arr.splice(j, 1);
            n--;  
            j--;  
        }
    }
}


console.log("Array after removing duplicates:", arr);
